JeffShepherd/Rancid-Tomatillos
# Feature/e2e testing

## What does this PR do?
- Installs Cypress testing framework
- Conducts end-to-end testing for the Rancid Tomatillos
- User flows include: 
  - clicking on a card link
  - navigating with Router
  - happy/sad path testing with network requests

## How should it be tested in the UI?
- install Cypress locally (` npm i -D cypress --save-dev`)
- In `package.json`, create the following script:
  `'cypress:open': 'cypress open'`
- run `npm run cypress:open`
- All tests should pass
